samples/apps/forum/test/demo/submit-opinions.ts

```diff
@@ -44,7 +44,7 @@ async function main() {
         }
         console.log('Submitting opinions for user ' + user)
         try {
-            await bent('PUT', 204)(`${ENDPOINT_URL}/all`, req, getAuth(jwt))
+            await bent('PUT', 204)(`${ENDPOINT_URL}`, req, getAuth(jwt))
```

Comment:
This caused the submission to yield a slightly funky validation error about unexpected opinions.